starter-code/test1.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="test1.css">
+    <title>Omar Salman | Portfolio</title>
+</head>
+<body>
+   <center>
+      <hr color= "black" size="6" width="80%" height ="30%">
+
+      <p><a href="file:///Users/hnanalshmry/code/wdi/hw-w02d01-html-and-css/starter-code/website2.html">About Me</a>|Portfolio</p>
```

Comment:
this link will only work on your computer you need to add a relative path not an absolute

Reply:
also, you need to link Portfolio